prediction_market_agent/agents/prophet_agent/deploy.py

```diff
@@ -81,3 +81,33 @@ def load(self) -> None:
         self.agent = OlasAgent(
             model=DEFAULT_OPENAI_MODEL, embedding_model=EmbeddingModel.openai
         )
+
+
+class DeployablePredictionProphetGPTo1PreviewAgent(DeployableTraderAgentER):
+    agent: PredictionProphetAgent
+
+    def load(self) -> None:
+        super().load()
+        # o1-preview supports only temperature=1.0
+        self.agent = PredictionProphetAgent(
+            model="o1-preview-2024-09-12",
```

Comment:
This one is pretty hungry:

```
$15.00 / 1M input tokens
$60.00 / 1M output tokens
```

Maybe we will have to shut it down or at least limit after first few days 😄